### DISCUSSION TITLE
Unable to Login As Super_Admin

### DISCUSSION THREAD

**Discussion by andrewdmaclean** [chatwoot/chatwoot#1384] [Self hosted] [answered]
**Describe the bug**
When attempting to connect to the super_admin URL on a heroku-hosted instance, I am unable to login with my credentials 

**To Reproduce**

1. Go to `XYZ.herokuapp.com/super_admin/sign_in`
2. Input Email & Password
4. See error - "Couldn't find SuperAdmin"

**Expected behavior**

Access to Super Admin Dashboard

**Screenshots**

![image](https://user-images.githubusercontent.com/26526271/98305835-733ed900-1f77-11eb-8c94-486f5120fd92.png)

**Environment**

Describe whether you are using Chatwoot Cloud (app.chatwoot.com) or a self hosted installation of Chatwoot. If you are using a self hosted installation of Chatwoot describe the type of deployment (Docker/Linux VM installation/Heroku)

**Desktop (please complete the following information):**
 - OS: Windows 10 Home
 - Browser: Firefox
 - Version: 82.0.2 (64-bit)

**Comment by sojan-official** [comment, marked as answer]
@andrewdmaclean  super admin uses separate credentials. 
you need to create a super admin user before accessing the page 

https://www.chatwoot.com/docs/self-hosted/monitoring/super-admin-sidekiq

**Reply by NitinNR** [reply]
Can we use this on linux self hosted service?

**Reply by sojan-official** [reply]
> Can we use this on linux self hosted service?

Yes you can

**Reply by ywptr** [reply]
the above wiki link leads to an empty page now. but the article to create super admin user is now here: https://www.chatwoot.com/docs/self-hosted/monitoring/super-admin-sidekiq

**Comment by andrewdmaclean** [comment]
@sojan-official that was it!

Thanks for the help 👌

**Comment by cruiservlad** [comment]
Hello! When I try to exec command from https://www.chatwoot.com/docs/self-hosted/monitoring/super-admin-sidekiq I got next message in console: 
```
3.0.2 :003 > SuperAdmin.where(:email => 'pisarevvlad@gmail.com').first_or_create(name: 'test', password: 'test')
(Object doesn't support #inspect)
```
What I do wrong?

**Reply by sojan-official** [reply]
@cruiservlad  The documentation is slightly outdated, you don't need to create super admin using the command anymore. You should be able to access the super admin interface using the admin credentials that you created while installing chatwoot
